patternfly/patternfly-elements
# feat: Add event emission to pfe-cta (#741)

Fixes #741 

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* https://github.com/patternfly/patternfly-elements/issues/741
* Demo page: https://deploy-preview-761--happy-galileo-ea79c4.netlify.com/elements/pfe-cta/demo/

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Automated tests should pass.
2. Click a link and ensure the console message appears on the demo page.

#### Browser requirements
Your component should work in all of the following environments:


### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
